lib/ansible/modules/database/mysql/mysql_user.py

```diff
@@ -605,9 +583,9 @@ def main():
             connect_timeout=dict(type='int', default=30),
             config_file=dict(type='path', default='~/.my.cnf'),
             sql_log_bin=dict(type='bool', default=True),
-            client_cert=dict(type='path', aliases=['ssl_cert']),
-            client_key=dict(type='path', aliases=['ssl_key']),
-            ca_cert=dict(type='path', aliases=['ssl_ca']),
+            ssl_cert=dict(type='path'),
```

Comment:
Not sure why these aliases have been swapped around?
This change will break people using `client_cert:` in their playbooks

Reply:
fixed